apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/API.java

```diff
@@ -176,14 +179,34 @@ public API(PropertiesHolder propertiesHolder, UPnP upnp, JettyConnectorCreator j
     }
 
     public static String findWebUiDir() {
-        String dir = DirProvider.getBinDir() + File.separator + WEB_UI_DIR;
-        dir = dir + File.separator + "build";
-        File res = new File(dir);
-        if (!res.exists()) { //we are in develop IDE or tests
-            dir = DirProvider.getBinDir() + "/apl-exec/target/" + WEB_UI_DIR + "/build";
-            res = new File(dir);
+        final Path binDir = DirProvider.getBinDir();
+
+        final Path htmlStubPath =
+            binDir.resolve("bin").resolve("html-stub").toAbsolutePath();
```

Comment:
html stub is mandatory?

Reply:
yes, it's used if web-ui is not available

Reply:
yes, it's used if web-ui is not available